My shows and Inputstream Adaptive fix

snoopyjoe opened the pull request:
Fix for episodes not loading in My Shows for some shows.
Also, a work around for streaming issues when Inputstream Adaptive is enabled. May be removed later if deemed not necessary or Inputstream Adaptive gets updated.

snoopyjoe committed:
My shows and Inputstream Adaptive fix

Fix for episodes not loading in My Shows for some shows.
Also, a work around for streaming issues when Inputstream Adaptive is enabled. May be removed later if deemed not necessary or Inputstream Adaptive gets updated.

eracknaphobia reviewed (changes requested):
Looks good other than the placement of the stream_url change.

snoopyjoe committed:
Inputstream Adaptive modification

Checks to see if Inputstream Adaptive is enabled and uses alternate stream to prevent crashing.
